bindings/go/main_test.go

```diff
@@ -122,12 +123,34 @@ func TestVerifyKZGProof(t *testing.T) {
 // Benchmarks
 ///////////////////////////////////////////////////////////////////////////////
 
+func GetRandFieldElement(seed int64) Bytes32 {
+	rand.Seed(seed)
+	bytes := make([]byte, 31)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		panic("failed to get random field element")
+	}
+
+	var fieldElementBytes Bytes32
+	copy(fieldElementBytes[:], bytes)
```

Comment:
I'm guessing this leaves the last byte as 0? Could be worth commenting.

Reply:
Yes, that's how it works. I'll add a comment 👍